website/src/components/EventSlider/index.tsx

```diff
@@ -72,6 +72,7 @@ const EventSlider: React.FC<EventsProps> = ({
         settings: {
           slidesToShow: 1,
           centerMode: true,
+          centerPadding: '30px',
```

Comment:
can we use theme.spacing here?

Reply:
As this won't come inside the `makeStyles` the `theme.spacing` will require adding another import of `useTheme` and value will be printed without unit
ex: theme.spacing(3.8) = 30.4, no units will be added.

I think its better if we add this directly @Pallavi-PH